rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -26,7 +26,6 @@
 import rocal_pybind as b
```

Comment:
in another PR, we should rename rocal_pybind to more meaningful name like rocal_backend or something.